src/prefect/cli/work_queue.py

```diff
@@ -626,3 +627,38 @@ async def delete(
     else:
         success_message = f"Successfully deleted work queue {name!r}"
     exit_with_success(success_message)
+
+
+@work_app.command("read-runs")
+@experimental_parameter("pool", group="work_pools", when=lambda y: y is not None)
+async def read_wq_runs(
```

Comment:
It'd be nice to have a couple tests around this.

Reply:
I added a few tests! Let me know if you think there are edge cases I should be testing for.

Reply:
That seems to hit all the notes, thank you!